django_tables2/rows.py

```diff
@@ -162,6 +165,25 @@ def _call_render(self, bound_column, value=None):
             'table': self._table,
         })
 
+    def get_cell_value(self, name):
+        '''
+        Returns the final rendered value (excluding any html) for a cell in the row, given the name
+        of a column.
+        '''
+        return self._get_and_render_with(name, self._call_value)
+
+    def _call_value(self, bound_column, value=None):
```

Comment:
not sure of this needs to be another method on `Rows`. For me, it gets confusing even though I'm quite familiar with the code base.
Can we dedup the _call_render/_call_value code?